lib/admin_page/user/mobile_and_web_layout_design/mobile_design.dart

```diff
@@ -0,0 +1,294 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:web/model/user_data.dart';
+
+import '../../../values/app_assets.dart';
+import '../widget/widget_user_detail.dart';
+
+class MobileLayout extends StatefulWidget {
+   MobileLayout({Key? key, this.user}) : super(key: key);
+  UserData? user;
+  @override
+  State<MobileLayout> createState() => _MobileLayoutState();
+}
+
+class _MobileLayoutState extends State<MobileLayout> {
+
+  @override
+  Widget build(BuildContext context) {
+    double height = MediaQuery.of(context).size.height;
+    double width = MediaQuery.of(context).size.width;
+    return Stack(
+      children: [
+        Text(
+          "+++$width++",
+          style: const TextStyle(
+              color: Colors.black, fontWeight: FontWeight.bold, fontSize: 16),
+        ),
+        Container(
+          decoration: const BoxDecoration(
+            image: DecorationImage(
+                image: AssetImage(AppAssets.backGroundImage),
+                fit: BoxFit.cover),
+          ),
+          child: Padding(
+            padding: EdgeInsets.only(
+                top: height * 0.14,
+                bottom: height * 0.14,
+                left: width * 0.1,
+                right: width * 0.1),
+            child: Container(
+              width: width,
+              height: height,
+              decoration: BoxDecoration(color: Colors.white, boxShadow: [
+                BoxShadow(
+                  offset: const Offset(0.5, 0.5),
+                  color: Colors.white.withOpacity(0.8),
+                ),
+              ]),
+              child: Stack(
+                children: [
+                  Column(
+                    children: [
+                      Text(
```

Comment:
Don't share this with production or live build if required wrap the same in debug mode condition. In case you are using the same in multiple places then it will be difficult to remove the same from everywhere.